src/dsp_tools/commands/validate_data/deserialise_input.py

```diff
@@ -129,7 +129,14 @@ def _deserialise_text_prop(prop: etree._Element) -> list[SimpleTextDeserialised
     for val in prop.iterchildren():
         match val.attrib["encoding"]:
             case "utf8":
-                all_vals.append(SimpleTextDeserialised(prop_name, val.text))
+                all_vals.append(SimpleTextDeserialised(prop_name, _get_text_as_string(val)))
             case "xml":
-                all_vals.append(RichtextDeserialised(prop_name, val.text))
+                all_vals.append(RichtextDeserialised(prop_name, _get_text_as_string(val)))
     return all_vals
+
+
+def _get_text_as_string(value: etree._Element) -> str | None:
+    if len(value):
+        return "".join(etree.tostring(child, encoding="unicode") for child in value)
```

Comment:
I think this could fail with nested tags. I guess you need something like `value.iterdescendants()`. But to be sure, it would be good to add some more test cases:

- no tags at all
- one tag at the start
- one tag in the middle
- several tags in the middle